src/chrome/content/rules/HangSengBank.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="Hang Seng Bank">
+	<target host="www.hangseng.com" />
+	<target host="bank.hangseng.com" />
```

Comment:
ebusiness.hangseng.com seems to work
Please also add a comment for the mismatch on www.cn.hangseng.com